django/db/migrations/operations/base.py

```diff
@@ -1,6 +1,17 @@
+import enum
+
 from django.db import router
 
 
+class OperationCategory(enum.Enum):
```

Comment:
Not sure what the best practices wrt/ to `Enum` are but since this hosts `str` should we extend `(str, enum.Enum)` instead?

Reply:
Nick Pope created [StrEnum](https://github.com/django/django/blob/c65f49d3cb8709f2f694f78b4849bc7693e90416/django/db/models/enums.py#L22) as well which would be slightly more convenient.